app/experimenter/nimbus-ui/src/components/Summary/TableBranches/index.test.tsx

```diff
@@ -150,15 +150,15 @@ describe("TableBranches", () => {
           treatmentBranches: [
             {
               name: "treatment",
-              slug: "",
+              slug: "treatment-1",
               description: "",
               ratio: 0,
               featureValue: null,
               featureEnabled: true,
             },
             {
               name: "",
-              slug: "",
+              slug: "treatment-2",
```

Comment:
Why change this one? I don't think you can have a slug set without a name set.

Reply:
You're right, and my change made the test invalid anyway, so I've updated it 👍